srv/src/main/resources/application.yaml

```diff
@@ -9,6 +9,7 @@ cds:
   odata-v4:
     endpoint.path: "/api"
   security:
+    authentication.normalize-provider-tenant: true
```

Comment:
Mhh, somehow I don't like this approach to add this configuration to make sed work. Maybe we change our ITest script here instead.

Reply:
I was thinking about this as well, but in a second step. I wanted to at least make the tests green again. Proposal: As we have dedicated tests which also tests tenant isolation ([link](https://github.wdf.sap.corp/cds-java/cds-services/blob/c9169f16569991ea7406c5540ef3d9badc4ddb7e/integration-tests/bookshop-cloud/src/test/java/com/sap/cds/demo/BookshopITest.java#L104)), we create a second subscriber account next to `bookshop-subscriber` and use this one instead of the provider account. This requires changes in several repos because of which I didn´t want to do this before getting tests green again at least once.

Reply:
Ok, I looked into the history of this file and we used already this approach in the past. This config parameter was removed recently. Let's add it again to get the test green. Afterwards we can rework the ITests and remove it again.